utils/utils.js

```diff
@@ -0,0 +1,11 @@
+const client = require('../config/connection');
+
+exports.internalServerError = (error) => {
+  return { status: 'error', message: error };
+};
+
+exports.getDataByField = async (tableName, fieldName, id) => {
+  const { rows } = await client.query(`SELECT * FROM ${tableName} WHERE ${fieldName} = '${id}'`);
```

Comment:
can you make this more inclusive, by using these [conditionals](https://www.javatpoint.com/postgresql-condition)